README.md

```diff
@@ -1,18 +1,27 @@
-# Краткая инструкция по работе
+# Work instructions 👩‍🎓👨‍🎓
+
+## To get started, must be installed: 🔔
 
-## Для начала работы должен быть установлен:
 - Node.js v.10.15.3
 - Gulp v4
 - npm last version
 
-## Основные команды для работы
-- Установка - npm i
-- Запуск локального сервера - npm start
-- Сборка проекта без запуска локального сервера - npm run build
-- Запуск тестирования на соответствия код-гайдам - npm test
+## Basic commands for work 💻
+
+- Installation - npm i
+- Running a local server - npm start
+- Running a project without starting a local server - npm run build
+- Running testing with code guides - npm test
```

Comment:
1. Replace title by "Available commands"
2. Emoji should be written with shortcode (https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)
3. Change order of text to be "[command] - Description", for example "npm start - Running a local server"
4. Wrap all commands in backtick for syntax highlighting (see https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code)
5. Replace `npm i` by `npm install` (better to write full command than short version because we are not sure everybody can understand it)